src/maptm/src/maptm_ovsdb.c

```diff
@@ -0,0 +1,312 @@
+        else if (!strcmp(ptr, "\"false\""))
+        {
+            return false;
+        }
+        ptr = strtok(NULL, ",");
+    }
+
+    return false;
+}
+
+// Node_Config callback
+void callback_Node_Config(
+        ovsdb_update_monitor_t *mon,
+        struct schema_Node_Config *old_rec,
+        struct schema_Node_Config *conf
+)
+{
+
+    if ((mon->mon_type) == OVSDB_UPDATE_NEW)
+    {
+        LOGD("%s: new node config entry: module %s, key: %s, value: %s",
+                __func__, conf->module, conf->key, conf->value);
+                
+        if (!strcmp(conf->module, "MAPTM"))
+        {
+            if (maptm_get_supportValue(conf->value))
+            {
+                WanConfig |= MAPTM_ELIGIBILITY_ENABLE;
+            }
+            strucWanConfig.mapt_support = maptm_get_supportValue(conf->value);
+            maptm_dhcp_option_update_15_option(strucWanConfig.mapt_support);
+            maptm_dhcp_option_update_95_option(strucWanConfig.mapt_support);        
+        }
+    }
+
+    if (mon->mon_type == OVSDB_UPDATE_DEL)
+    {
+        LOGD("%s: node config entry deleted: module %s, key: %s, value: %s",
+                __func__, old_rec->module, old_rec->key, old_rec->value);
+    }
+
+    if (mon->mon_type == OVSDB_UPDATE_MODIFY)
+    {
+        LOGD(   "%s: node config entry updated: \n"
+                "old module: %s, old key: %s, old value: %s \n"
+                "new module: %s, new key: %s, new value: %s",
+                __func__, old_rec->module, old_rec->key, old_rec->value,
+                conf->module, conf->key, conf->value);
+                
+        if (!strcmp(conf->module, "WANO"))
+        {
+            strucWanConfig.mapt_support = maptm_get_supportValue(conf->value);
+            if (maptm_get_supportValue(conf->value) != maptm_get_supportValue(old_rec->value))
+            {
+                maptm_dhcp_option_update_15_option(strucWanConfig.mapt_support);
+                maptm_dhcp_option_update_95_option(strucWanConfig.mapt_support);     
+            }
+            if ((maptm_get_supportValue(conf->value)) && !maptm_get_supportValue(old_rec->value))
+            {
+                WanConfig |= MAPTM_ELIGIBILITY_ENABLE;
+            }
+            else if (!maptm_get_supportValue(conf->value) && (maptm_get_supportValue(old_rec->value)))
+            {
+                WanConfig &= MAPTM_IPV6_ENABLE;
+            }
+            if (maptm_get_supportValue(conf->value) != maptm_get_supportValue(old_rec->value))
+            {
+                maptm_eligibilityStart(WanConfig);
+                osp_ps_t *ps = NULL;
+                ps = osp_ps_open("MAPT_SUPPORT", OSP_PS_RDWR | OSP_PS_PRESERVE);
+                if (ps == NULL)
+                {
+                    LOG(ERR, "Error saving new MAP-T support value");
+                }
+                if (!(osp_ps_set(ps, "MAPT_SUPPORT", wano_mapt_get_supportValue(conf->value) ? "true" : "false",sizeof(wano_mapt_get_supportValue(conf->value)))))
```

Comment:
Function `wano_mapt_get_supportValue()` is not defined, this will fail to compile/link.
Did you mean `mapt_get_supportValue()`?